src/Eto/Forms/ThemedControls/ThemedDocumentControlHandler.cs

```diff
@@ -47,16 +71,152 @@ public Font Font
 		}
 	}
 
+	/// <summary>
+	/// Gets or sets the disabled foreground color.
+	/// </summary>
+	/// <value>The disabled foreground color.</value>
+	public Color DisabledForegroundColor
+	{
+		get { return disabledForegroundColor; }
+		set
+		{
+			disabledForegroundColor = value;
+			Calculate();
```

Comment:
We don't need to `Calculate()` after setting a color as it does not affect any of the sizes being calculated.

Reply:
Applied.

Reply:
Looks like you went a little to far and also removed this call in `HasNavigationButtons` setter. That one actually influences calculations.